quantuminspire/util/configuration.py

```diff
@@ -155,10 +156,43 @@ def store_tokens(self, host: Url, tokens: TokenInfo) -> None:
         :param tokens: OAuth access and refresh tokens
         :return: None
 
-        This functions stores the access and refresh tokens in the config.json file.
+        This function stores the member_id, access and refresh tokens in the config.json file.
         """
         self.auths[host].tokens = tokens
+        self.auths[host].team_member_id = self._get_team_member_id(host, tokens.access_token)
         assert isinstance(self.model_config["json_file"], PathLike)
         Path(self.model_config["json_file"]).write_text(
             self.model_dump_json(indent=2), encoding=self.model_config.get("env_file_encoding")
         )
+
+    def _get_team_member_id(self, host: Url, access_token: str) -> int:
+        """
+
+        :param host: the hostname of the api for which the tokens are intended
+        :param access_token: OAuth access token for the api
+        :return: int
+
+        This function fetches the member_id to be used for creating a project.
+        """
+
+        url_members = f"{host}/members"
+        response = requests.get(url_members, headers={"Authorization": "Bearer " + access_token})
```

Comment:
Shouldn't we do this via the compute-api-client?

Reply:
I changed it to using compute api client now.